test/addons/dlopen-hello-world/test.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const os = require('os');
+process.dlopen(module, require.resolve(`./build/${common.buildType}/binding`),
+               os.constants.dlopen.RTLD_NOW | os.constants.dlopen.RTLD_LOCAL)
```

Comment:
Wasn't an accident, I was kind of relying on that behavior to avoid skipping the test. Yeah, the alternative sounds like a good idea.